packages/ui-react/src/components/Button/Button.tsx

```diff
@@ -29,6 +29,7 @@ type Props = {
   busy?: boolean;
   id?: string;
   activeClassname?: string;
+  navLinkState?: any;
```

Comment:
We are using `strict` mode, so `any` is not allowed 😃

Reply:
`NavLink` from React Router has a prop `state`. Its type is `any`. It's the only reason I would ever use `any` like this. :)

Reply:
Yeah, but I believe running `yarn lint:ts` will fail when using `any`, right? Probably need to change it to `unknown`

Reply:
It doesn't fail.